### DISCUSSION TITLE
Which installer for Debian 11

### DISCUSSION THREAD

**Discussion by willw-carallon** [abraunegg/onedrive#1681] [General]
Am looking at installing this on a Debian 11 machine and would prefer to use a prebuilt package if possible.

From the docs it seems like I should be using a package from the opensuse build service. However the package for Debian 10 doesn't work:

> The following packages have unmet dependencies:
> onedrive : Depends: libphobos2-ldc-shared82 (>= 1:1.12.0) but it is not installable

Should I be using Debian Testing? That could well have been for bullseye before it became the stable release. Does it matter that the build service says it is unresolvable?

Thanks
Will

**Comment by abraunegg** [comment]
@willw-carallon 
Whilst the OpenSuSE Build Service provides for Debian 10, it currently needs to be uplifted for Debian 11.

If you refer to https://packages.debian.org/search?keywords=onedrive - you can see that v2.4.13 is in the Debian Testing repository, vs v2.4.10 which is in the Debian main repository.

My advice here is to use the Debian Testing repository for your package or build from source.

@norbusan 
Are you able to add Debian 11 to your OpenSuSE Build Service?

**Reply by willw-carallon** [reply]
Thanks. Never occured for me to look in debian package repository, I thought the table in INSTALL.md suggested not. I will use the package from debiena testing.

**Reply by abraunegg** [reply]
> Thanks. Never occured for me to look in debian package repository, I thought the table in INSTALL.md suggested not. I will use the package from debiena testing.

It is recommended to use the OpenSuSE Build Service, because the packages are more current than the Debian default ones - but now that Debian 11 has been released, this kind of changes things up a little.

Once the OpenSuSE Build Service has been updated to support Debian 11, the recommendation will still be to use that, rather than the default packages they will get stale when the next release is done.

**Comment by norbusan** [comment]
Hi @abraunegg 
I added `Debian_11` to the build list, see https://build.opensuse.org/project/show/home:npreining:debian-ubuntu-onedrive for the complete list of packages. This should now cover most areas ...

**Comment by abraunegg** [comment]
@norbusan 
Awesome - thankyou :)

**Comment by abraunegg** [comment]
@willw-carallon 
Documentation is updated to reflect Debian 11 install process using the OpenSuSE Build Service packages.
